File: Java/nopackage/classobject/Integer.java
public class Integer {
    private int i;

    public Integer() {
        i = 0;
    }

    public Integer(int i) {
        this.i = i;
    }

    Integer plus(Integer x) {
        Integer sum = new Integer();
        sum.i = this.i + x.i;
        return sum;
    }

    int get() {
        return this.i;
    }

    public static void main(String[] args) {
        Integer a = new Integer(10);
        Integer b = new Integer(20);
        Integer c;
        c = a.plus(b);
        System.out.println("c = " + c.get());
    }
}

File: Python/class-object/integer.py
class Integer:
    i = 0

    def __init__(self, i):
        self.i = i

    def plus(self, x):
        sum = Integer(0)
        sum.i = self.i + x.i
        return sum
    
    def get(self):
        return self.i
    

a = Integer(10)
b = Integer(20)
c = a.plus(b)
print(f"c = {c.get()}")